db/seeds.rb

```diff
@@ -253,7 +253,7 @@
 
     article = Article.create!(
       body_markdown: markdown,
-      featured: true,
+      featured: i.zero?, # only feature the first article,
```

Comment:
Sneaking this in to adjust seed file to be a little more realistic.

Reply:
Especially since we use squash and merge rather than rebase and merge, I'd prefer to keep this vastly different change separate (I guess a different PR, since the commits get vanished) from the infra-specific stuff I think - just makes it easier to `git blame` or `git bisect` through history later if it's ever necessary.